packages/core/src/command/patient-import/steps/result/patient-import-result-local.ts

```
@@ -42,7 +44,7 @@ export class PatientImportResultLocal implements PatientImportResult {
     } catch (error) {
       const msg = `Failure while processing job result @ PatientImport`;
       log(`${msg}. Cause: ${errorToString(error)}`);
-      await updateJobAtApi({ cxId, jobId, status: "failed" });
+      await updateJobAtApi({ cxId, jobId, status: "failed", throwOnFailure: false });
```

This is needed so we don't flag the error of updating the API as the first error.

---

is this approach preferred to try/catch in this block? It'd seem preferable to leave the core method as is and just handle the case were we don't want to re-throw this error here, but maybe there's a preference to the implemented pattern instead.